src/styles/_utilities_custom.scss

```diff
@@ -0,0 +1,17 @@
+@import "bootstrap/scss/functions";
+@import "bootstrap/scss/variables";
+@import "bootstrap/scss/maps";
+@import "bootstrap/scss/mixins";
+@import "bootstrap/scss/utilities";
+
+$utilities: map-merge($utilities,
+    (
+      "background": (property: background-color,
+        class: "bg",
+        values: (
+            'brand': #5B2487,
+        )),
+    ));
+
+
+@import "bootstrap/scss/utilities/api";
```

Comment:
This was just to generate a new utility class, which we don't actually use now, so the whole file can be removed. I expect we'll want to us this API down the line, but no need introducing it until we have to.